dashboard/models.py

```diff
@@ -205,7 +205,7 @@ class Course(models.Model):
     GRADING_CHOICES = [('Percent', 'Percent'), ('Point', 'Point'), ]
     show_grade_type = models.CharField(verbose_name="Show Grade Type", max_length=255,
                                          choices=GRADING_CHOICES, default='Percent')
-    data_last_updated = models.DateTimeField(verbose_name="Time data for this course was last updated", null=True, blank=True)
+    data_last_updated = models.DateTimeField(verbose_name="When this course was last updated", null=True, blank=True)
```

Comment:
Maybe this doesn't need a verbose name, and we can just use `data_last_updated`? I think it would be nice if we referred to this field consistently everywhere. Above in `admin.py` you use "course updated date". If you want to include more info, we could add `help_text="When associated resource data was last updated for this course"`. That language also seems more accurate, since I think the course model data is always updated if it changes.

These model changes may also need a migration, though I still don't understand why Django requires these when they don't result in database changes.

Reply:
I'm saying we need a consistent way to refer to this field. If you use that long verbose name that doesn't match what is used in the admin action you added, it may not be clear to the user what that admin action does. Thus, I'm saying use "data last updated" everywhere, and then add help text to explain to the user what it is.

Reply:
Are you talking about consistency between this and 

`@admin.action(description='Clear selected Courses updated dates')`

I'm not sure how specific we need to be to an admin using the UI. This was the date that this course was last updated by the **cron** for all data, not just resource data.

Reply:
Yes, consistency between these. You refer to this field three ways:
- `verbose_name="When this course was last updated"` (which shows up as a column header)
- `@admin.action(description='Clear selected Courses updated dates')` (which shows up in the dropdown)
- `self.message_user(request, "All selected course updated dates are cleared."` (which shows up in the success message).

Maybe these are close enough to where it doesn't matter, but if this was facing all users, I'd want it to be more standardized. Just being consistent would help people know how they are all related. If a short field name like "Data last updated" isn't clear enough, that's where help text may be beneficial.

I think you are correct, it's not just resource data. But it's not just course data either, it's data related to the course too. So "courses updated dates" or "course updated dates" isn't exactly the full picture.

Reply:
Thanks, I'll review these suggestions and try to make these more consistent. The only one I'm really adding is the `admin.action` here but the name is more exposed now if we add it to the table so might as well. I think we could create an issue to review some of these descriptions, maybe with @jennlove-um and see if any could be improved or could use additional `help_text`.

Reply:
Okay, I'll go with your suggestion for the verbose_name and build from there

```
verbose_name="Data last updated" (which shows up as a column header)

@admin.action(description='Clear selected last updated values') (which shows up in the dropdown)

self.message_user(request, "Selected last updated values cleared." (which shows up in the success message).
```

Reply:
You may not need the `verbose_name` value at all, it may default to that based on the field name.

Reply:
Right, It looks like the default name takes from the variable name. So right it probably isn't necessary if we just want data_last_updated -> "Data last updated"

Reply:
It looks like we have redundant verbose names for many fields already even in this table like "Term", "Name", "Show Grade Type". I think I'll just leave it explicit and consistent rather than remove. I didn't know that it expanded the field name as the verbose_name.

Reply:
I added the migration you mentioned that seemed to be needed for just this `verbose_name` changing.